### ISSUE TITLE
Track number vocalization:

### ISSUE THREAD

**Issue by ranetto** [jcsteh/osara#345]
Hello I would like to propose to put a check in the preferences of osara to set whether or not it should say the track number when scrolling through the tracks.
In jaws this was implemented at my request by the snowman, but when I use NVDA it would be nice to be able to make sure that Osara doesn't announce the number of tracks as when you have a lot of tracks and you name them all right, the number is just a nuisance limits the speed of scrolling between them.
Thanks in advance to James Scott and all of you.

**Comment by ScottChesworth** [comment]
Trouble is, we can't make it a universal change. Even if OSARA stops reporting track numbers in the TCP, they'd still be reported in the combo boxes located in the I/O window, third context menu for tracks etc. Has Jim scripted his way around that, or just settled for the UX in the TCP being different? So far as I understand it, the former would be a ton of hackery, and I can't say I'd be a fan of the latter. Consistency is the key to productivity for a lot of people.

Are you already aware of the widely used tricks to speed up navigation? First two that spring to mind are my custom actions in Dropbox for jumping to the master, the first and the last track. There's also SWS actions to jump to nearest previous and nearest next folders that @nofishonfridays made for us.

**Comment by ranetto** [comment]
Thanks Scott, you already use all the actions to go from folder to folder and I have also created some to go to the first and last track, as you say it is not very consistent that in the various views the track numbers would still be displayed , but since it is only a check in the osara preferences, who would use it would know in advance what the new behavior refers to: it could be called announce track number
 in the "TCP". I think that in the other lists you mentioned it is not so important to go very fast, as in the "TCP" where maybe you are recording a synth bass two or three tracks of an electric bass and you want to go quickly because the customer asks you to add some 'bass or because you have an acoustic that you have to dub but in the center you also have its center dub.
I use it with snowman scripts when using jaws and I am very happy with it.
When you are in the "i / o", or in the send maybe even if you hear the track number it can help you differently from the "TCP" not to get distracted and therefore it doesn't slow you down very much.
Excuse me Scott, maybe I particularly feel this need because recording a lot of commercial pop, as you know there are dozens of tracks of vocals, choirs, acoustic and electronic drums and much more.
I thank you in advance.

**Comment by ScottChesworth** [comment]
We've all done high track counts here man. Aside from the lumpy UX, my question is this: is not hearing a single number really gonna save a significant enough amount of time to justify the effort it'd take to implement the feature? That's not for me to decide, but I think it is a valid consideration. Over to you, @jcsteh.

**Comment by ranetto** [comment]
Hi Scott and James, I don't know how the numbers and trace names are displayed by the reaper bees, if in a single segmented and indexed string or if in two values one numeric and one string or how, on the screen there is the number in a type of label and the name in a different label and therefore I don't know how long it would take to implement this function and if James will find it interesting and important. To me it would be a lot but it is clear that if this involved a lot of work to extract this information from the string that will be sent to speech I would be very sorry to have to add additional work to James.
However, I thank you for your concern.

**Comment by betov75** [comment]
I use a regular expression inside the NVDA speech dictionary to remove them. It may be a simple workaround. It doesn't affect anything else in OSARA form me since I never use any kind of enumeration.

**Comment by jcsteh** [comment]
This is a duplicate of #175, so I'm closing this. @ranetto, it'd be great if you could provide answers to my questions in that issue, since the original requester of this feature never got to them. Thanks.
